llvm/lib/Target/AIE/AIEWawRegRewriter.cpp

```diff
@@ -94,8 +94,7 @@ class AIEWawRegRewriter : public MachineFunctionPass {
   // uses. These physical register should not be used for replacement
   // candidates, since this would introduce new WAW dependencies, which this
   // pass removes.
-  SmallSet<MCPhysReg, 16>
-  getMBBForbiddenReplacements(const MachineBasicBlock *MBB);
+  SmallSet<MCPhysReg, 16> getDefBlockedPhysRegs(const MachineBasicBlock *MBB);
```

Comment:
Super-nit: "defBlocked" is a bit unclear. Maybe just `getDefinedPhysRegs`?

Reply:
updated the name